src/pages/library/DataBrowserPreview.js

```diff
@@ -139,9 +152,14 @@ const DataBrowserPreview = ({ id }) => {
             isSearchable: true,
             isClearable: false,
             value: selectedTable,
-            getOptionLabel: ({ value }) => `${_.startCase(value)}${tableMap[value].rowCount === 0 ? ' (0 Rows)' : ''}`,
+            getOptionLabel: ({ rowCount, value }) => {
+              return div({ style: { color: colors.dark(rowCount ? 1 : 0.5) } }, [_.startCase(value)])
+            },
+            formatGroupLabel: ({ label }) => {
+              return div({ style: { marginTop: 5, paddingTop: 15, borderTop: `1px solid ${colors.dark(0.5)}`, color: colors.dark(0.8) } }, label)
```

Comment:
Arrays for children plzz
```suggestion
              return div({ style: { marginTop: 5, paddingTop: 15, borderTop: `1px solid ${colors.dark(0.5)}`, color: colors.dark(0.8) } }, [label])
```